openapi.yaml

```diff
@@ -50,7 +52,81 @@ tags:
     description: Operations to manage users in Service Workbench
   - name: Cost
     description: Operation to view cost breakdown
+  - name: authN
+    description: Operation for login, logout, authN provider
 paths:
+  /api/authentication/provider/configs:
+    get:
+      tags:
+        - admin
+        - authN
+      summary: Get authentication provider configs detail
+      operationId: GetAuthNConfigsDetail
+      description: |
+        Get authentication provider configs detail
+      responses:
+        '200':
+          description: Array of authN provider configs detail
```

Comment:
I noticed in the rest of the API docs, the description has the word "return" in the beginning of the description. Do we want to continue that pattern? For example in this case it would be: 
`Return array of authN provider configs detail`

Reply:
Good question! Just spot checked the doc, looks like we already have both, with or without `Return`. I'll leave it as is for now.